webpack/webpack.common.js

```diff
@@ -18,7 +18,7 @@ const getGlobalConstantsFile = (env) => {
     if (fs.existsSync(envSpecificPath)) {
         return envSpecificPath;
     }
-    return path.resolve(__dirname, '../sampleGlobalConstants_local.js');
+    return path.resolve(__dirname, '../GlobalConstants.js');
 };
```

Comment:
only update is to reflect the rename of `sampleGlobalConstants_local` to `GlobalConstants`